@here/harp-vectortile-datasource/test/resources/geoJsonData.ts

```diff
@@ -104,7 +104,7 @@ export const THEME: Theme = {
                     size: 8,
                     priority: 100
                 },
-                renderOrder: 10.3
+                renderOrder: 10
             }
```

Comment:
maybe I overlooked it, but I think there is no test for the introduced asserts if renderOrder is no integer, might be good to add  a test for it

Reply:
I believe we have many styles were the render order is not an integer. Is there a way to avoid to use bigint in this or?